src/alignment/mod.rs

```diff
@@ -216,7 +217,66 @@ impl Alignment {
             }
         }
 
-        format!("{}\n{}\n{}", x_pretty, inb_pretty, y_pretty)
+        let mut s = String::new();
```

Comment:
Can you provide a motivation for this block in contrast to the one-liner before. I guess it formats differently, but ffrom the code I just cannot see immediately from the code.